gradio/components/gallery.py

```diff
@@ -31,24 +32,30 @@
 if TYPE_CHECKING:
     from gradio.components import Timer
 
-GalleryImageType = Union[np.ndarray, PIL.Image.Image, Path, str]
-CaptionedGalleryImageType = Tuple[GalleryImageType, str]
+GalleryMediaType = Union[np.ndarray, PIL.Image.Image, Path, str]
+CaptionedGalleryMediaType = Tuple[GalleryMediaType, str]
 
 
 class GalleryImage(GradioModel):
     image: FileData
     caption: Optional[str] = None
 
 
+class GalleryVideo(GradioModel):
+    video: FileData
+    caption: Optional[str] = None
+    subtitles: Optional[FileData] = None
```

Comment:
Do we support subtitles in videos in a Gallery? How would a user pass that in?

Reply:
Oh yeah will remove, there's no way to pass it in as of now. We can look into adding them later.